cloudinit/config/cc_chef.py

```diff
@@ -50,7 +56,7 @@
     "client_key": "/etc/chef/client.pem",
     "json_attribs": CHEF_FB_PATH,
     "file_cache_path": "/var/chef/cache",
-    "file_backup_path": "/var/backups/chef",
+    "file_backup_path": "/var/chef/backup",
```

Comment:
Have you verified that these newer directories are also used on Focal?

Reply:
Verified through omnibus install on focal, that default directories are `/var/chef/backup` and `/var/chef/cache` nowadays even on focal's packaged chef version 15.8.25.